helm/cluster/templates/clusterapi/controlplane/_helpers_clusterconfiguration_apiserver.tpl

```diff
@@ -33,8 +35,10 @@ extraArgs:
   feature-gates: {{ include "cluster.internal.controlPlane.kubeadm.clusterConfiguration.apiServer.featureGates" $ }}
   {{- end }}
   kubelet-preferred-address-types: InternalIP
+  {{- if $.Values.internal.advancedConfiguration.controlPlane.apiServer.enablePriorityAndFairness }}
   max-mutating-requests-inflight: "${MAX_MUTATING_REQUESTS_INFLIGHT}"
```

Comment:
what is the actual value and where does it come from?
is kubeadm somehow dynamically calculating this?